### ISSUE TITLE
Add block pushing to Planner

### ISSUE THREAD

**Issue by lotu** [ccplanner/planner#8]
Add ability to push blocks to planner.  Can use sokoban code from benchmarks.  Blocks pushed into water become dirt.  Blocks can be pushed along thin walls and over gravel.  They can also be used to trigger buttons.  They can defuse bombs by blowing them up.  Bombs will need to be added to do this.

**Comment by darrenks** [comment]
basic block pushing is now enabled, but stuff like bombs, switch, and ice aren't implemented with them yet. also gravel is not implemented.

**Comment by lotu** [comment]
Is there anything left for block pushing?

**Comment by darrenks** [comment]
yes, but these possibly non classical things still need to be handled

block cloning
dynamic blocks (not sure what those actually are)
pushing blocks onto ice
running into a block while sliding
holding a switch
